src/couch/src/couch_httpd_rewrite.erl

```diff
@@ -438,7 +438,7 @@ path_to_list([<<>>|R], Acc, DotDotCount) ->
 path_to_list([<<"*">>|R], Acc, DotDotCount) ->
     path_to_list(R, [?MATCH_ALL|Acc], DotDotCount);
 path_to_list([<<"..">>|R], Acc, DotDotCount) when DotDotCount == 2 ->
-    case config:get("httpd", "secure_rewrites", "true") of
+    case chttpd_util:get_chttpd_config(secure_rewrites, "true") of
```

Comment:
This could be a `get_boolean/3` I think. The setting might have been created before we had a `config:get_boolean/3` I think.